corehq/util/urlsanitize/ip_resolver.py

```diff
@@ -1,19 +1,33 @@
-from operator import itemgetter
-from socket import getaddrinfo
+# from operator import itemgetter
+# from socket import getaddrinfo, gaierror
+import socket
+import ipaddress
 
 
-def sort_ip_addresses(ip_addresses):
-    return sorted(ip_addresses, key=itemgetter(0))
+# def sort_ip_addresses(ip_addresses):
+#     return sorted(ip_addresses, key=itemgetter(0))
+
+# def resolve_ip(address, port=80):
+#     ips = sort_ip_addresses(socket.getaddrinfo(address, port))
+#     return extract_ip(ips[0])
```

Comment:
No need to keep commented-out code. It's already in git. You can always get it back if you want it. e.g.
```
$ git show 425b899d:corehq/util/urlsanitize/ip_resolver.py
```

Reply:
yikes, sorry. I thought I thoroughly checked everything I was committing, but apparently I missed this